tests/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ set(
     0012-produce_consume.c
     0013-null-msgs.c
     0014-reconsume-191.c
+    0016-client_swname.c
```

Comment:
tip: if you use your toes as well as your fingers, you can count up this high.

Reply:
(buf if you're too lazy too, 16 comes after 15)

Reply:
damnit